raiden/messages.py

```diff
@@ -502,21 +516,22 @@ class Secret(EnvelopeMessage):
 
     def __init__(
             self,
-            message_identifier,
-            payment_identifier,
-            nonce,
-            token_network_address,
-            channel,
-            transferred_amount,
-            locked_amount,
-            locksroot,
-            secret,
+            chain_id: ChainID,
+            message_identifier: MessageID,
+            payment_identifier: PaymentID,
+            nonce: int,
+            token_network_address: Address,
+            channel_identifier: ChannelID,
+            transferred_amount: int,
+            locked_amount: int,
```

Comment:
`transferred_amount` and `locked_amount` should be a `TokenAmount` in all occurrences in this file